schemas/browsers.schema.json

```diff
@@ -76,6 +76,10 @@
           "type": "boolean",
           "description": "Whether the browser supports user-toggleable flags that enable or disable features."
         },
+        "accepts_webdriver_bidi": {
```

Comment:
Sorry if this has been discussed before but for my understanding and for later reference, could you elaborate why this "accepts" info is needed and why it is specifically for BiDi (as opposed to say `accepts_webdriver`)?

Reply:
It's used in the `lint/linter/test-browsers-presence.ts` to ensure that there are entries for all browsers supporting WebDriver BiDi.

But you make a good point that this could (probably) be generalized to `accepts_webdriver`.

Reply:
I attempted the change in 3f79f8bbc, but this caused a linting failure:

```
> @mdn/browser-compat-data@5.6.42 lint
> tsx lint/lint.ts

Loading and checking files...
Testing browser data...
Testing feature data...
Testing all features together...
Browser Presence - 62 problems (62 errors, 0 warnings):
 ✖ webdriver.commands.AcceptAlert - Error → Has the following browsers, which are invalid for webdriver compat data: ie, oculus, opera, opera_android, samsunginternet_android
 ✖ webdriver.commands.AddCookie - Error → Has the following browsers, which are invalid for webdriver compat data: ie, oculus, opera, opera_android, samsunginternet_android
```

In doubt, we could simply omit adjusting the tests for now.

Reply:
OK just to spell it out: It seems the purpose of this additional validation is to forbid ie, oculus, opera, opera_android, samsunginternet_android from having webdriver (BiDi) data at all. If that's a desirable and needed goal here, then let's introduce the additional lining. 

I think it would be cleaner to have `accepts_webdriver` but I don't want to block this further and/or make you do even more work. So, I guess either we land as is or we split out this linting part from this PR altogether and land it later with the changes to the classic data.

Reply:
I have backed out the changes in [4dae5fa](https://github.com/mdn/browser-compat-data/pull/25804/commits/4dae5fa2470ba8b0493c7e8098a55e38c4283f22) for now, so we can add them later separately.